backend/events/apps.py

```diff
@@ -85,6 +85,24 @@ def on_tuples(cc_event, block_number, tx_id, tx_status):
                 queue=worker_queue
             )
 
+        if 'ComputePlans' in payload:
+            handle_compute_plan_events(payload['ComputePlans'])
+
+
+def handle_compute_plan_events(cps):
+    for cp in cps:
+        handle_compute_plan_event(cp)
+
+
+def handle_compute_plan_event(cp):
+    id = cp['computePlanID']
```

Comment:
be aware that id is a builtin callable in Python and here you override it.
```python
print(id(1))
id=1
print(id(1))
```

Reply:
wow, I had no idea, thanks!

Reply:
Fixed: https://github.com/SubstraFoundation/substra-backend/pull/90/commits/14804e7fd2ff97ad2ac2599bed5bcb4ce6a01baa